packages/next-codemod/transforms/__testfixtures__/next-async-request-api-dynamic-apis/async-prop-cast-01.input.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { 
+  headers,
+  cookies, 
+  draftMode, 
+} from 'next/headers'
+
+export function MyDraftComponent() {
+if (draftMode().isEnabled) {
+    return null
+  }
```

Comment:
seems like we'd benefit from running these tsx files through prettier

Reply:
updated in [60c89bd](https://github.com/vercel/next.js/pull/69572/commits/60c89bd001710349bcbb92d11beb80c9f3cff814)